mindjunkies/lecture/tests.py

```
@@ -1,98 +1,71 @@
-from django.core.files.uploadedfile import SimpleUploadedFile
-from django.test import TestCase
+import pytest
 from django.utils.text import slugify
-
+from django.contrib.auth import get_user_model
```

Ruff (0.8.2)

`django.contrib.auth.get_user_model` imported but unused

Remove unused import: `django.contrib.auth.get_user_model`

(F401)